variants/STM32F7xx/F745Z(E-G)T_F746Z(E-G)(T-Y)_F750Z8T_F756ZG(T-Y)/variant_ETHERCAT_DUINO.h

```diff
@@ -0,0 +1,303 @@
+#define PA1_ALT1                (PA1  | ALT1)
+#define PA1_ALT2                (PA1  | ALT2)
+#define PA2_ALT1                (PA2  | ALT1)
+#define PA2_ALT2                (PA2  | ALT2)
+#define PA3_ALT1                (PA3  | ALT1)
+#define PA3_ALT2                (PA3  | ALT2)
+#define PA4_ALT1                (PA4  | ALT1)
+#define PA5_ALT1                (PA5  | ALT1)
+#define PA6_ALT1                (PA6  | ALT1)
+#define PA7_ALT1                (PA7  | ALT1)
+#define PA7_ALT2                (PA7  | ALT2)
+#define PA7_ALT3                (PA7  | ALT3)
+#define PA15_ALT1               (PA15 | ALT1)
+#define PB3_ALT1                (PB3  | ALT1)
+#define PB8_ALT1                (PB8  | ALT1)
+#define PB9_ALT1                (PB9  | ALT1)
+#define PB14_ALT1               (PB14 | ALT1)
+#define PB14_ALT2               (PB14 | ALT2)
+#define PC0_ALT1                (PC0  | ALT1)
+#define PC0_ALT2                (PC0  | ALT2)
+#define PC1_ALT1                (PC1  | ALT1)
+#define PC1_ALT2                (PC1  | ALT2)
+#define PC2_ALT1                (PC2  | ALT1)
+#define PC2_ALT2                (PC2  | ALT2)
+#define PC3_ALT1                (PC3  | ALT1)
+#define PC3_ALT2                (PC3  | ALT2)
+#define PC4_ALT1                (PC4  | ALT1)
+#define PC5_ALT1                (PC5  | ALT1)
+#define PC6_ALT1                (PC6  | ALT1)
+#define PC7_ALT1                (PC7  | ALT1)
+#define PC8_ALT1                (PC8  | ALT1)
+#define PC9_ALT1                (PC9  | ALT1)
+#define PC10_ALT1               (PC10 | ALT1)
+#define PC11_ALT1               (PC11 | ALT1)
+
+#define NUM_DIGITAL_PINS        91
+#define NUM_ANALOG_INPUTS       16
+
+#define CANRX                   PD0
+#define CANTX                   PD1
+
+// EtherCAT controller pins (not broken out)
+#define ECAT_SYNC0              PF0
+#define ECAT_SYNC1              PF1
+#define ECAT_IRQ                PF2
+#define ECAT_RST                PF5
+#define ECAT_D3                 PF6
+#define ECAT_D2                 PF7
+#define ECAT_D0                 PF8
+#define ECAT_D1                 PF9
+#define ECAT_SCK                PB2
+#define ECAT_LED_RUN            PB4
+#define ECAT_LED_ERR            PB5
+#define ECAT_CS                 PB6
+
+// Serial pins
+#define TX0                     PC10          // UART4      
+#define RX0                     PC11          // UART4     
+#define TX0_ALT                 PC10_ALT1     // UART3 
+#define RX0_ALT                 PC11_ALT1     // UART3
+
+#define TX1                     PE8           // UART7
+#define RX1                     PE7           // UART7
+
+#define TX2                     PB10          // UART3
+#define RX2                     PB11          // UART3
+
+#define TX3                     PD5           // UART2
+#define RX3                     PD6           // UART2
+
+// I2C pins
+#define SDA0                    PF15          // I2C4_ALT
+#define SCL0                    PF14          // I2C4_ALT
```

Comment:
Why ALT ?

```suggestion
#define SDA0                    PF15          // I2C4
#define SCL0                    PF14          // I2C4
```

Reply:
ALT was something I called it in my testing firmware, so that is no longer relevant